roletracker/roletracker.py

```diff
@@ -0,0 +1,253 @@
+import logging
+import hashlib
+import asyncio
+import discord
+from datetime import datetime
+import time
+
+from redbot.core import checks, commands, Config, modlog
+from redbot.core.bot import Red
+from redbot.core.config import Group
+from redbot.core.commands import Context, Cog
+from redbot.core.utils.chat_formatting import pagify
+from redbot.core.utils.predicates import MessagePredicate
+
+try:
+    from redbot.core.commands import GuildContext
+except ImportError:
+    from redbot.core.commands import Context as GuildContext
+
+__author__ = "TheBluekr#2702"
+__cogname__ = "aurelia.cogs.roletracker"
+
+
+class RoleTracker(commands.Cog):
+    def __init__(self, bot):
+        super().__init__()
+        self.bot = bot
+        self.logger = logging.getLogger(__cogname__)
+        self.config = Config.get_conf(self, identifier=428038701, force_registration=True)
+
+        default_role = {"addable": False, "USERS": {}}
+
+        self.config.register_role(**default_role)
+
+    async def initialize(self):
+        await self.register_casetypes()
+
+    @staticmethod
+    async def register_casetypes():
+        # register mod case
+        role_case = {
+            "name": "Role List Update",
+            "default_setting": True,
+            "image": "\N{PAGE FACING UP}",
+            "case_str": "roleupdate",
```

Comment:
this is what appears when case is created, so case_str should be "Role Update" or something like that, and name should be "roleupdate".